server/api/sites/[siteId]/stories/[storyId]/translated-slugs/index.put.ts

```
@@ -0,0 +1,76 @@
+    .delete(storyTranslatedSlugs)
+    .where(
+      and(
+        eq(storyTranslatedSlugs.storyId, storyId),
+        eq(storyTranslatedSlugs.siteId, siteId),
+        eq(storyTranslatedSlugs.organisationId, secure.organisationId)
+      )
+    )
+
+  // Skip empty values and insert new translations
+  const slugsToInsert = langEntries
+    .filter(([_, slug]) => slug && slug.trim())
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)